fe/fe-core/src/main/java/org/apache/doris/catalog/external/ExternalTable.java

```diff
@@ -103,7 +105,16 @@ public boolean isView() {
     }
 
     protected void makeSureInitialized() {
-        throw new NotImplementedException();
+        try {
+            // getDbOrAnalysisException will call makeSureInitialized in ExternalCatalog.
+            ExternalDatabase db = catalog.getDbOrAnalysisException(dbName);
+            if (db.isInitialized()) {
+                return;
+            }
+            db.makeSureInitialized();
```

Comment:
I think we can move `db.isInitialized()` into `db.makeSureInitialized()`. Because `db.makeSureInitialized();` is a reentryable method